benchmark-service/deployment.yaml

```diff
@@ -32,6 +32,8 @@ spec:
             value: /deploy-config/deploy-config.json
           - name: HAIL_SHA
             value: "{{ code.sha }}"
+          - name: HAIL_CI_OAUTH_TOKEN
+            value: /secrets/oauth-token/oauth-token
```

Comment:
I don't think you need the second `oauth-token`. Is that what CI did?

Reply:
Do you need this? I think the default path should be the one for the deployment rather than the local one. Then you can get rid of this.

Reply:
don't think I need it but then what should I replace this with
with open(os.environ.get('HAIL_CI_OAUTH_TOKEN', 'oauth-token/oauth-token'), 'r') as f:

Reply:
I believe you can replace the default path with `/secrets/oauth-token/oauth-token`. I looked at CI and it looks like they kept the environment variable. Up to you I guess.